homeassistant/components/wevolor/config_flow.py

```diff
@@ -0,0 +1,72 @@
+"""Config flow for Wevolor Control for Levolor Motorized Blinds integration."""
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from pywevolor import Wevolor
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.exceptions import HomeAssistantError
+
+from .const import CONFIG_CHANNELS, CONFIG_HOST, CONFIG_TILT, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONFIG_HOST): str,
+        vol.Required(CONFIG_TILT, default=False): bool,
+        vol.Required(CONFIG_CHANNELS, default=6): vol.All(int, vol.Range(min=1, max=6)),
+    }
+)
+
+
+async def validate_input(hass: HomeAssistant, data: dict[str, Any]) -> dict[str, Any]:
+    """Validate the user input allows us to connect."""
+
+    wevolor = Wevolor(data[CONFIG_HOST])
+    status = await wevolor.get_status()
+
+    if not status:
+        raise CannotConnect
+
+    return status
+
+
+class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for Wevolor Control for Levolor Motorized Blinds."""
+
+    VERSION = 1
+
+    async def async_step_user(
+        self, user_input: dict[str, Any] | None = None
+    ) -> FlowResult:
+        """Handle the initial step."""
+        if user_input is None:
+            return self.async_show_form(
+                step_id="user", data_schema=STEP_USER_DATA_SCHEMA
+            )
+
+        errors = {}
+
+        try:
+            info = await validate_input(self.hass, user_input)
+        except CannotConnect:
+            errors["base"] = "cannot_connect"
+        except Exception as err:  # pylint: disable=broad-except
+            _LOGGER.exception("Unexpected exception: %s", err)
+            errors["base"] = "unknown"
```

Comment:
`Exception`-test is missing. 100% test coverage is required for `config_flow.py`.
You can check your coverage with:
```
pytest ./tests/components/wevolor --cov=homeassistant.components.wevolor --cov-report term-missing -vv
```
Other files can be added to `.coveragerc` to ignore the test coverage.